test/e2e/nstemplatetier_test.go

```diff
@@ -153,54 +153,70 @@ func TestUpdateNSTemplateTier(t *testing.T) {
 	hostAwait := awaitilities.Host()
 	memberAwait := awaitilities.Member1()
 
-	// first group of users: the "cheesecake lovers"
-	cheesecakeSyncIndexes := setupAccounts(t, awaitilities, "cheesecake", "cheesecakelover%02d", memberAwait, count)
-	// second group of users: the "cookie lovers"
-	cookieSyncIndexes := setupAccounts(t, awaitilities, "cookie", "cookielover%02d", memberAwait, count)
-
-	// setup chocolate tier to be used for creating spaces without usersignups
-	spaces := setupSpaces(t, awaitilities, "chocolate", "chocolatelover%02d", memberAwait, count)
-
-	cheesecakeSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cheesecakeSyncIndexes, "cheesecake", "base", "base", true)
-	cookieSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cookieSyncIndexes, "cookie", "base", "base", true)
-	verifyResourceUpdatesForSpaces(t, awaitilities, memberAwait, spaces, "chocolate", "base", "base")
-
-	// when updating the "cheesecakeTier" tier with the "advanced" template refs for namespaces (ie, same number of namespaces) but keep the ClusterResources refs
-	updateTemplateTier(t, hostAwait, "cheesecake", "advanced", "")
-	// and when updating the "cookie" tier with the "baseextendedidling" template refs (ie, same number of namespaces)
-	updateTemplateTier(t, hostAwait, "cookie", "baseextendedidling", "baseextendedidling")
-	// and when updating the "chocolate" tier to the "advanced" template for Namespace refs
-	updateTemplateTier(t, hostAwait, "chocolate", "advanced", "")
-
-	// then
-	cheesecakeSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cheesecakeSyncIndexes, "cheesecake", "advanced", "base", false)
-	cookieSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cookieSyncIndexes, "cookie", "baseextendedidling", "baseextendedidling", false)
-	verifyResourceUpdatesForSpaces(t, awaitilities, memberAwait, spaces, "chocolate", "advanced", "base")
+	// we will have a lot of usersignups who are affected by the tier updates, so
+	// we need to increase the timeouts on assertions/awaitilities to allow for all resources to be updated
+	hostAwaitTimeout := hostAwait.Timeout
+	memberAwaitTimeout := memberAwait.Timeout
+	defer func() {
+		// restore original timeouts
+		hostAwait.Timeout = hostAwaitTimeout
+		memberAwait.Timeout = memberAwaitTimeout
+	}()
+	hostAwait.Timeout = hostAwait.Timeout + time.Second*time.Duration(3*count*2)     // 3 batches of `count` accounts, with 2s of interval between each update
+	memberAwait.Timeout = memberAwait.Timeout + time.Second*time.Duration(3*count*2) // 3 batches of `count` accounts, with 2s of interval between each update
```

Comment:
Correct me if I'm wrong, but couldn't this be simplified to
```suggestion
hostAwait = hostAwait.WithRetryOptions(TimeoutOption(hostAwait.Timeout + time.Second*time.Duration(3*count*2)))       // 3 batches of `count` accounts, with 2s of interval between each update
	memberAwait = memberAwait.WithRetryOptions(TimeoutOption(memberAwait.Timeout + time.Second*time.Duration(3*count*2))) // 3 batches of `count` accounts, with 2s of interval between each update
```

Reply:
I did not remember about the `WithRetryOptions`, but I believe we should still reset the timeouts to their initial value, shouldn't we?

Reply:
oh, ok, then it's much simpler, indeed